src/ZODB/scripts/repozo.py

```diff
@@ -755,6 +746,88 @@ def do_recover(options):
             raise
 
 
+def do_incremental_recover(options, repofiles):
+    datfile = os.path.splitext(repofiles[0])[0] + '.dat'
+    log('Recovering (incrementally) file to %s', options.output)
+    with open(datfile) as fp, open(options.output, 'r+b') as outfp:
+        outfp.seek(0, 2)
+        initial_length = outfp.tell()
+        previous_chunk = None
+        for line in fp:
+            fn, startpos, endpos, _ = chunk = line.split()
+            startpos = int(startpos)
+            endpos = int(endpos)
+            if endpos > initial_length:
+                break
+            previous_chunk = chunk
+        else:
+            if endpos == initial_length:
+                log('Target file is same size as latest backup, '
+                    'doing nothing.')
+                return
+            else:
+                log('Target file is longer than latest backup, '
```

Comment:
Is it correct to use "longer" / "shorter" and not "larger" / "smaller" ?

Reply:
Thanks, I edited to use "larger" / "smaller". I think this comes from the calls to .seek() and .tell(), which makes the file looks like a long line, and not like an object.